Flutter/installation/linux-installer/how-to-download.md

```diff
@@ -28,12 +28,12 @@ Our 30-day trial can be downloaded in two ways.
 ### Download Free Trial Setup
 
 1. You can evaluate our 30-day free trial by visiting the [Download Free Trial](https://www.syncfusion.com/downloads) page and select the product
-2. After completing the required form or logging in with your registered Syncfusion account, you can download the trial installer from the confirmation page. (as shown in below screenshot.)
+2. After completing the required form or logging in with your registered Syncfusion<sup>&reg;</sup> account, you can download the trial installer from the confirmation page. (as shown in below screenshot.)
 
    ![Trial and downloads of Syncfusion Flutter](images/trial-confirmation.png)
```

Comment:
Hi @AswiniDileep,

Trademark is not added in this hyperlink. Check and ensure it once.

Regards,
Hari Hara Sudhan. K.

Reply:
Hi @sfHariHaraSudhan,

As it is an image file not added as this text changes won't show as other text in preview else in web after changes also.
It is just hyperlink for image. 

Regards,
Aswini S.